openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -83,7 +84,7 @@ def create_mock_profile(self, user):
         legacy_profile = UserProfile.objects.get(id=user.id)
         legacy_profile.country = "US"
         legacy_profile.level_of_education = "m"
-        legacy_profile.year_of_birth = 1900
+        legacy_profile.year_of_birth = 2000
         legacy_profile.goals = "world peace"
         legacy_profile.mailing_address = "Park Ave"
         legacy_profile.bio = "Tired mother of twins"
```

Comment:
Make sure to add year_of_birth and profile_image to the read-only field test on line 411.